MantidPlot/src/Mantid/MantidDock.cpp

```diff
@@ -631,19 +635,24 @@ void MantidDockWidget::addPeaksWorkspaceMenuItems(QMenu *menu, const Mantid::API
 }
 
 /**
-* Add the actions that are appropriate for a MatrixWorkspace
+* Add the actions that are appropriate for a WorkspaceGroup
 * @param menu :: The menu to store the items
+* @param groupWS :: [input] Workspace group related to the menu
 */
-void MantidDockWidget::addWorkspaceGroupMenuItems(QMenu *menu) const
-{
+void MantidDockWidget::addWorkspaceGroupMenuItems(
+    QMenu *menu, const WorkspaceGroup_const_sptr &groupWS) const {
   m_plotSpec->setEnabled(true);
   menu->addAction(m_plotSpec);
   m_plotSpecErr->setEnabled(true);
   menu->addAction(m_plotSpecErr);
   menu->addAction(m_colorFill);
   m_colorFill->setEnabled(true);
-  menu->addAction(m_plotSurface);
-  m_plotSurface->setEnabled(true);
+  if (groupWS && groupWS->getNumberOfEntries() > 2) {
```

Comment:
2 workspaces produces an empty surface plot - needs to have 3 or more to show anything in the  plot window